src/components/BooksList.js

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 
 // import Book from './Book';
```

Comment:
[OPTIONAL] Please remove the commented code to keep your code clean.

Reply:
Well noted. Will work on it. 
Thanks :smile: